docs/user-guide/widgets.md

```diff
@@ -214,35 +214,35 @@ An example of counter widget could be:
 
 ## Manage existing widgets
 
-Once widgets have been created, they will be placed on the bottom right of the map with three buttons available on their top (**Pin**, **Collapse** and **Menu**) and the *Widgets Tray* appears:
+Once widgets have been created, they will be placed on the bottom right of the map viewer and the *Widgets Tray* appears:
 
-<img src="../img/widgets/widgets-tray.jpg" class="ms-docimage"/>
+<img src="../img/widgets/widgets-tray.jpg" class="ms-docimage" style="max-width:500px;"/>
 
-The user is now allowed to menage widgets performing the following operations:
+Through the buttons available on each widget the user can  perform the following operations:
 
-* Move and resize them
+<img src="../img/widgets/widget-button.jpg" class="ms-docimage" style="max-width:500px;"/>
 
-* Fix and expand/collapse them
-
-* Access their menu
-
-### Move and resize a widget
-
-In order to move a widget you can simply drag and drop it inside the viewport, and in order to resize it you can click in the lower right corner and drag it to the desired size:
+* Drag and drop the widget to move it within the viewer and **resize** it thought the <img src="../img/button/resize-button.jpg" class="ms-docbutton"/> button (also available for widgets present in a dashboard)
 
 <img src="../img/widgets/ded-widgets.gif" class="ms-docimage" style="max-width:500px;"/>
 
-### Fix and expand/collapse a widget
+* **Fix** the position and the dimension of the widget thought the <img src="../img/button/pin.jpg" class="ms-docbutton"/> button
```

Comment:
* **Pin** the position and the dimension of the widget thought the <img src="../img/button/pin.jpg" class="ms-docbutton"/> button